src/content.ts

```diff
@@ -29,8 +29,8 @@ if (checks()) {
 
 	// disable default sorting library
 	const script = document.createElement('script')
-	script.innerHTML = `$('#sortable_nav').sortable('destroy'); $('#sortable_nav').enableSelection();`
-	document.head.appendChild(script)
+	script.src = chrome.runtime.getURL('public/csp.js') ;
+	(document.head || document.documentElement).appendChild(script);
```

Comment:
Since `csp.js` is a separate file, I think `content.ts` is doing too many things.

We can remove this disabling business out of `content.ts` and keep this as a content script instead of injecting.

`csp.js` will inject the code instead, which can be something like this:

~~~js
const disablingCode = ```$('#sortable_nav').sortable('destroy');
$('#sortable_nav').enableSelection();
```
const script = document.createElement('script');
script.textContent = disablingCode;
(document.head || document.documentElement).appendChild(script);
~~~

Reply:
This should be a separate pull request though.